### ISSUE TITLE
[The Halls of Reflection]

### ISSUE THREAD

**Issue by Goodhank** [dalaranwow/dalaran-wow#5331]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I cannot loot the subdued Quel'Delar in the [Halls of Reflection] quest as a part of the Battered Hilt quest chain. If I bring others along they can loot the corpse but I cannot. Additionally the loot is non-tradeable so it doesnt help me if they can loot it.

I have tried relogging, resetting instance, landing the killing blow, being the primary source of damage, changing my graphics. All to no avail.

**Expected behaviour**: The item should be lootable upon killing the mob.

**Steps to reproduce the problem**:

1.  Obtain [Battered Hilt]
2. Reach [The Halls of Reflection] quest in Battered Hilt questchain
3. Defeat the sword and attempt to loot the corpse. 

**Include proofs for this behaviour**

https://youtu.be/fJe0YHt3nbc

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Goodhank** [comment]
After running the dungeon on heroic, the item was finally lootable. Thank you.

**Comment by Rushor** [comment]
You need to do this quest on heroic mode
